### PR TITLE
Move the android mobile CI to actions.

### PR DESCRIPTION
**Pull request by Jorropo**

### CODE REVIEW

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -1,55 +0,0 @@
-steps:
```
Comment (moul) [inline .buildkite/pipeline.yml:1]: keep buildkite for the moment, we need to build on both for some days to check that everything is ok

File: .github/workflows/android.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+jobs:
+  build:
+    name: Build for Android
+    runs-on: ubuntu-latest
+    steps:
+      - name: Mount bazel cache
+        uses: actions/cache@v1
+        with:
+          path: "/home/runner/.cache/bazel"
+          key: bazel
```
Comment (moul) [inline .github/workflows/android.yml:21]: we don't use bazel anymore

File: .buildkite/pipeline.yml (hunk 1 above)
Comment (Jorropo) [reply .buildkite/pipeline.yml:1]: ok

File: .github/workflows/android.yml (hunk 2 above)
Comment (Jorropo) [reply .github/workflows/android.yml:21]: ah ok, this was on the install script of buildkite :) (I havn't finished yet)

**Comment by Jorropo** [comment]
We could speed up the IOS build using [Carthage](https://github.com/Carthage/Carthage) to cache some xcode stuff but I don't think that worth the effort (that something build framework like fastlane).

**Review by moul — APPROVED** [review]

**Pull request merged**